api/v1/models/Event.js

```diff
@@ -8,12 +8,9 @@ var Event = Model.extend({
     idAttribute: 'id',
     validations: {
         name: ['required', 'string', 'maxLength:255'],
-        shortName: ['required', 'string', 'maxLength:25'],
         description: ['required', 'string', 'maxLength:2047'],
-        tracking: ['required', 'boolean'],
-        startTime: ['required', events.verifyDate],
-        endTime: ['required', events.verifyDate],
-        tag: ['required', 'string', events.verifyTag]
```

Comment:
I could see `tag` being useful if it's used properly. For example, if something is `PRE_EVENT` vs `POST_EVENT` or something like that. Do you think it should be removed?